src/resmom/requests.c

```diff
@@ -4576,19 +4556,16 @@ post_restart(job *pjob, int ev)
 			}
 		}
 
-		new_stime = time_now - (ckpt_time - pjob->ji_qs.ji_stime);
```

Comment:
This snippet of code has been taken out and it takes into account the checkpoint time (ckpt_time)...

Reply:
`ckpt_time` is used to calculate the length of time that the job was held. The whole logic is not necessary anymore so I removed all of them.